build.sc

```diff
@@ -60,6 +59,11 @@ class PluginModule(millBinaryVersion: String)
   def ivyDeps = super.ivyDeps() ++ Agg(
     ivy"com.lihaoyi::mill-scalalib:${millVersion(millBinaryVersion)}"
   )
+  override def sources = T.sources {
+    super.sources() ++ Seq(
+      millSourcePath / s"src-mill${millVersion(millBinaryVersion).split('.').take(2).mkString(".")}",
+    ).map(PathRef(_))
+  }
```

Comment:
You can avoid this since you don't seem to have Mill version-specific folders.

Reply:
Thanks! Removed it to avoid the clutter.
It correctly generates `0.9`, `0.10` and the milestone versions fine.